### ISSUE TITLE
Input Capture can not use the clipboard

### ISSUE THREAD

**Issue by sithlord48** [flatpak/xdg-desktop-portal#1434]
### Operating System

Any

### XDG Desktop Portal version

1.18

### Desktop Environment

Other

### Expected Behavior

it was discussed in https://github.com/flatpak/xdg-desktop-portal/pull/852 how the input capture would use the clipboard as well. 
Like many ppl I was expecting it to work in apps like input-leap but it does not seam to work.

### Current Behavior

Can not copy and paste between input-leap server and client when one is using wayland.

### Steps to Reproduce

1. setup input-leap on gnome 46+ or plasma 6.1
2. run a server / client for input -leap
3. try to copy and paste from server to client or client to server.

**Comment by swick** [comment]
Is support for clipboard sharing implemented in the client software, and x-d-p version you're tasting on? Why do you believe that this is a x-d-p issue?

**Comment by jadahl** [comment]
It's about the input capture portal allowing hooking up to the "abstract" clipboard portal. It's currently only usable via the remote desktop portal.

**Comment by swick** [comment]
Makes sense but is a feature request so I think this is better suited as a discussion?
